packages/postgres-database/src/simcore_postgres_database/models/file_meta_data.py

```diff
@@ -38,4 +31,7 @@
         doc="If true, this file is a soft link."
         "i.e. is another entry with the same object_name",
     ),
+    sa.Column(
+        "upload_expires_at", sa.DateTime(), nullable=True, doc="Timestamp of expiration"
```

Comment:
so since we create a presigned link to upload, this presigned link is created with a time limit. once that time limit is reached the presigned link loses functionality (one cannot use it anymore). And then the table should be cleaned up.